payloads/v8/_interactions/autocomplete.ts

```diff
@@ -0,0 +1,25 @@
+import type { APIBaseInteraction, ApplicationCommandOptionType, InteractionType } from '..';
+
+export type APIApplicationCommandAutocompleteInteraction = APIBaseInteraction<
+	InteractionType.ApplicationCommandAutocomplete,
+	APICommandAutocompleteInteractionData
+>;
+
+export interface APICommandAutocompleteInteractionData {
```

Comment:
This is not at all what data you receive from this interaction event 🤔
Check the notion document posted by devsnek in Discord Developer's (stage-info channel)